axelrod/tournament.py

```diff
@@ -70,6 +70,14 @@ def play(self, build_results=True, filename=None):
         """
         Plays the tournament and passes the results to the ResultSet class
 
+        Parameters
+        ----------
+
```

Comment:
spacing, and Parameters is capitalized (not sure what the guide is on this but the one above is changed)